src/guide/admonitions.rst

```diff
@@ -0,0 +1,58 @@
+Admonitions
```

Comment:
I understand providing a guide for when to use each type of admonition, but each one looks identical to all the others.  It seems like ``.. note::`` produces the same result as ``.. admonition:: Note``.  

Is there a way to apply different styles the admonitions?  Something like a red, thicker border to a Warning, for example?